ush/setup.sh

```diff
@@ -1622,14 +1622,15 @@ fi
 #
 #-----------------------------------------------------------------------
 #
-# Make sure that DO_ENSEMBLE is set to a valid value.  Then set the names
+# Make sure that DO_ENSEMBLE and DO_ENSEMBLE_REDUCED is set to a valid value.  Then set the names
 # of the ensemble members.  These will be used to set the ensemble member
```

Comment:
What does DO_ENSEMBLE_REDUCED do?

Reply:
This is to decide whether to run a subset of the ensemble forecast, for example, run 9 member free forecast (different from the 30 member EnKF cycling)